src/firefly/java/edu/caltech/ipac/firefly/server/query/UwsJobProcessor.java

```diff
@@ -307,7 +308,7 @@ public static Phase getPhase(String jobUrl) throws DataAccessException {
         }
     }
 
-    public static String getError(String jobUrl)  {
+    public static JobInfo.Error getError(String jobUrl)  {
         String errorUrl = jobUrl + "/error";
         HttpServices.Status status = HttpServices.getWithAuth(errorUrl, method -> {
```

Comment:
We should use the same logic as above.  Return the errorSummary if exists; otherwise return the message from `/error`.  If you want to remove the function because it's not used, that's fine by me as well.

Reply:
Yes it has a single use in L143. Are you saying, I can just move that conditional operator check (if errorSummary else /error endpoint) here?